packages/marko-web-html-sitemap/templates/date-list.marko

```diff
@@ -0,0 +1,60 @@
+$ const { config, GAM } = out.global;
+$ const { nodes, displayType, month, year, alias } = input;
+
+$ const type = "sitemap";
+$ const pageNode = {
+  title: 'Sitemap',
+};
+$ switch (displayType) {
+  case "day":
+      pageNode.description = `Select from the list of days in ${month} ${year}`;
```

Comment:
All {siteName} stories published in {month} {year}.

Reply:
@solocommand update this to the following: 
 - All Equipment World stories published by years
 - All Equipment World stories published by months in 2003
 - All Equipment World stories published by days in November 2003
 - All Equipment World stories published on November 26, 2003